packages/commerce-sdk-react/src/hooks/ShopperBaskets/cache.ts

```diff
@@ -322,6 +322,16 @@ export const cacheUpdateMatrix: CacheUpdateMatrix<Client> = {
             ]
         }
     },
+    updateItemsInBasket(customerId, {parameters}, response) {
+        return {
+            update: [
+                {queryKey: getBasket.queryKey(parameters)},
+                ...(customerId
+                    ? [updateCustomerBasketsQuery(customerId, parameters, response)]
+                    : [])
+            ]
```

Comment:
double checking, does the `updateItemsInBasket` endpoint returns same response structure as the getBasket query?

Reply:
The docs state "Success, the response body contains the basket with the updated item." Under the response section.

Reply:
Yes, I believe so. I also checked the return type in the `commerce-sdk-isomorphic` and both the `getBasket` and the `updateItemsInBasket` methods return a `Promise<Basket>`